src/frontend/qt_sdl/VideoSettingsDialog.h

```diff
@@ -74,9 +74,12 @@ private slots:
 private:
     void setVsyncControlEnable(bool hasOGL);
     void setEnabled();
+    int getsupportedRenderers();
 
     Ui::VideoSettingsDialog* ui;
     EmuInstance* emuInstance;
+    bool base_gl;
```

Comment:
These should probably be `baseGl` and `computeGl`, to match the camel case used throughout the codebase in such cases.

Reply:
I have seen multiple example of this case being used in the same codebase.